src/sentry/api/invite_helper.py

```diff
@@ -108,6 +114,13 @@ def handle_member_already_exists(self):
                 extra={"organization_id": self.om.organization.id, "user_id": self.request.user.id},
             )
 
+    def handle_member_has_no_SSO(self):
```

Comment:
```suggestion
    def handle_member_has_no_sso(self):
```
nit, but python method names don't typically have uppercase letters in them.

Reply:
Good point 😅